pytorch_lightning/utilities/distributed.py

```diff
@@ -18,7 +18,7 @@ def wrapped_fn(*args, **kwargs):
     # add the attribute to the function but don't overwrite in case Trainer has already set it
     getattr(rank_zero_only, 'rank')
 except AttributeError:
-    rank_zero_only.rank = os.environ.get('LOCAL_RANK', 0)
+    rank_zero_only.rank = int(os.environ.get('LOCAL_RANK', 0))
```

Comment:
While we're at it, could we simplify the try-catch block to
`rank_zero_only.rank = getattr(rank_zero_only, 'rank', int(os.environ.get('LOCAL_RANK', 0)))`
?